test/issue121-test.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable no-octal */
+// vim: expandtab:ts=2:sw=2
+
+const
+  assert = require('assert'),
+  assertions = require('./assertions'),
+  childProcess = require('./child-process').childProcess,
+  signals = ['SIGINT', 'SIGTERM'];
+
+describe('tmp', function () {
+  describe('issue121 - clean up on terminating signals', function () {
+    for (var i=0; i<signals.length; i++) {
+      it(signals[i], issue121Tests(signals[i]));
+    }
+  });
+});
+
+function issue121Tests(signal) {
+  return function (done) {
+    childProcess('issue121.json', function (err, stderr, stdout) {
+      if (err) return done(err);
+      else if (stderr) return done(new Error(stderr));
```

Comment:
I would remove the `else` keywords since the early returns simplify it:

```javascript
if (err) return done(err);
if (stderr) return done(new Error(stderr));

assertions.assertDoesNotExist(stdout);
done();
```

Reply:
There are multiple such cases. Let's do that prior to 0.1.0 release, what say ye?

Reply:
aarrrrrh laddies and lads, and a bottle of rum
ye need clean up the deck or it's the plank

Reply:
Back to normal again: this will all be cleaned up with #161 where I will fix the existing issues with async tests.